prac_06/programming_language.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 | Prac_06 | Dannielle Jones
+Class for a programming language.
+"""
+
+
+class ProgrammingLanguage:
+
+    def __init__(self, field="", typing="", reflection="", year=0):
+        self.field = field
```

Comment:
"field" is a heading for the fields in a table, not intended to be the "name" of the language.